pkg/render/monitor/monitor.go

```diff
@@ -743,7 +743,13 @@ func (mc *monitorComponent) serviceMonitorCalicoNode() *monitoringv1.ServiceMoni
 			Labels:    map[string]string{"team": "network-operators"},
 		},
 		Spec: monitoringv1.ServiceMonitorSpec{
-			Selector:          metav1.LabelSelector{MatchLabels: map[string]string{"k8s-app": "calico-node"}},
+			Selector: metav1.LabelSelector{
+				MatchExpressions: []metav1.LabelSelectorRequirement{
+					{Key: "k8s-app",
```

Comment:
The indentation here looks a little odd to me. I would expect it to look more like this:

```go
{
  Key: "k8s-app",
  Operator: metav1.LabelSelectorOpIn,
  Values:   []string{"calico-node", "calico-node-windows"},
},
```

Reply:
Yes, it's definitely weird, fill fix